### ISSUE TITLE
Sleep assinging crash , Seems to only happen with newly printet dupes

### ISSUE THREAD

**Issue by Dknir** [ihoromi4/ONIMods#11]
https://pastebin.com/QXQ25src

Only happened now when most of the cycle was already done and i took on another dupe (Homedweller abe in the log in this case)

**Comment by mgergye** [comment]
+1, can confirm, experiencing the same exception in: "(Minion).SleepChoreMonitor.root.checkforbed"
(for me the bug appeared only after loading a save, then hitting the first sleep cycle)

**Comment by CardiacTasty** [comment]
I've run into this bug as well, quite unfortunate.
